### PR TITLE
Fix the templatecache file name

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by rbovard** [comment]
Typos in commit messages:
* Fix the wsgi.co**n**f file name
* Fix the en**t**ry point

**Comment by sbrunner** [comment]
Thanks @rbovard 
This pullrequest is ready for a review

File: c2cgeoportal/scaffolds/create/+dot+gitignore_tmpl [hunk 1]
```diff
@@ -22,10 +22,11 @@
 /mapserver/c2cgeoportal.map
 /mapserver/tinyows.xml
 /deploy/hooks/pre-restore-database
-/{{package}}/static/build/
-/{{package}}/static-ngeo/build/
 /{{package}}/locale/*/LC_MESSAGES/*.mo
 /{{package}}/locale/{{package}}-*.pot
+/{{package}}/static-ngeo/build/
+/{{package}}/static-ngeo/fonts/gmf-icons.*
```
Comment (ger-benjamin) [inline c2cgeoportal/scaffolds/create/+dot+gitignore_tmpl]: Do you really want to ignore that ?

**Comment by ger-benjamin** [comment]
Except my comment and a need of rebase, this seems alright.

**Pull request merged**